course_catalog/models.py

```diff
@@ -1,26 +1,39 @@
 """
 course_catalog models
 """
+from django.utils import timezone
 from django.db import models
 from django.contrib.postgres.fields import JSONField
 
 
-class CourseInstructor(models.Model):
+class TimestampedModel(models.Model):
```

Comment:
Can you make the fields identical to this [TimestampedModel](https://github.com/mitodl/open-discussions/blob/master/open_discussions/models.py#L44-L54) in the discussions repo? These models will be imported into that project.

```python
    created_on = DateTimeField(auto_now_add=True)
    updated_on = DateTimeField(auto_now=True) 
```

Reply:
Made the change in commit #da4697b